transports/dns/src/lib.rs

```diff
@@ -777,7 +777,7 @@ mod tests {
             // type record lookups may not work with the system DNS resolver.
             let config = ResolverConfig::quad9();
             let opts = ResolverOpts::default();
-            let rt = tokio_crate::runtime::Builder::new_current_thread()
+            let rt = ::tokio::runtime::Builder::new_current_thread()
```

Comment:
nit:
```suggestion
            let rt = tokio::runtime::Builder::new_current_thread()
```
should work no?

Reply:
No because `super::*` is imported, and it contains module `tokio`.

Reply:
thanks for clarifying